kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -207,14 +208,34 @@ server {
 
             os.rename("/srv/storage/" .. tmp_file_name, "/srv/storage/" .. header_path_with_filename)
             os.remove("/srv/storage/" .. tmp_file_name .. ".shactx")
+
+            -- Send confirmation to the conductor after uploaded file.
+            local http = require "resty.http"
+            local httpc = http.new()
+            local res, err = httpc:request_uri("{{ upload_report_url }}" .. header_path_with_filename, {
```

Comment:
`res` is `result` or `response`? Please use full names.

`err` -> `conductor_error` or `upload_report_error`
`res` -> `conductor_response` or `upload_report_response`

Reply:
According to https://github.com/pintsized/lua-resty-http#request, `res` is the result of successful request, so also include error responses. Do we want to change the names to those you proposed ?

Reply:
Yes, if they're correct.

Reply:
Done.